### ISSUE TITLE
[Bug]: Error: spawn E2BIG

### ISSUE THREAD

**Issue by r-menezes** [denolehov/obsidian-git#450]
### Describe the bug

When creating a backup, obsidian-git aborts with the error "Error: spawn E2BIG".

### Relevant errors (if available)

```shell
Uncaught (in promise) Error: Error: spawn E2BIG plugin:  obsidian-git:21550
    at GitExecutorChain.onFatalException (plugin:obsidian-git:21550:79)
    at GitExecutorChain.eval (plugin:obsidian-git:21542:24)
    at Generator.throw (<anonymous>)
    at rejected (plugin:obsidian-git:20415:29)
onFatalException	@plugin:obsidian-git:21550
eval	@plugin:obsidian-git:21542
rejected	@plugin:obsidian-git:20415
```

### Steps to reproduce

I am unsure how to reproduce this error. I created and deleted many files (~2000) because I  imported a lot of unuseful data from Notion and then this error appeared.

### Operating system

Linux

### Installation Method

AppImage

### Plugin version

2.17.0

**Comment by Vinzent03** [comment]
Please share your plugin's config under advanced

**Comment by r-menezes** [comment]
thank you for the fast reply.

I managed to solve the issue by committing and pushing using the terminal. Creating backups with obsidian-git now gives no issues. I think it would be possible to close this issue.

All my Advanced configurations are the default, the fields are all empty with the default text showing:
![image](https://user-images.githubusercontent.com/38216046/222481642-4e75b0a0-1de6-45a5-aae6-b68b8dba0cd4.png)

For reference, I think this might have been caused by the configuration:

"List filenames affected by commit in the commit body": True

... perhaps is not a good idea to try to write a commit with thousands of lines?
